newsroom/clients/clients.py

```diff
@@ -0,0 +1,31 @@
+from content_api import MONGO_PREFIX
+
+import newsroom
+
+
+class ClientResource(newsroom.Resource):
+    """
+    Client schema
+    """
+
+    schema = {
+        "name": {
+            "type": "string",
+            "required": True,
+            "unique": True,
+        },
+        "password": {"type": "string", "required": True},
+    }
+
+    datasource = {
+        'source': 'clients',
+        'default_sort': [('name', 1)]
+    }
+    item_methods = ['GET', 'PATCH', 'DELETE']
+    resource_methods = ['GET', 'POST']
+    mongo_prefix = MONGO_PREFIX
+    internal_resource = True
```

Comment:
why is that internal resource? we need the api, and there would be no need for those views